src/domain/audio/AudioInput.ts

```diff
@@ -258,7 +255,13 @@ class AudioInput {
             sampleRate: AudioConstants.SAMPLE_RATE
         });
 
-        this.#_audioStreamSource = this.#_audioContext.createMediaStreamSource(this.#_audioInput);
+        try {
+            this.#_audioStreamSource = this.#_audioContext.createMediaStreamSource(this.#_audioInput);
+        } catch {
+            this.#_audioContext = new AudioContext();
+            this.#_audioStreamSource = this.#_audioContext.createMediaStreamSource(this.#_audioInput);
+        }
+
```

Comment:
Please add a comment on what this code does, i.e., if a browser doesn't automatically handle sample rate conversion then create an audio context at the hardware rate and handle the sample rate conversion for mic input in the audio worklet.
> Uncaught (in promise) DOMException: AudioContext.createMediaStreamSource: Connecting AudioNodes from AudioContexts with different sample-rate is currently not supported.

Aside; I wonder why sample rate conversion isn't required for audio output - unless that just happens to be 24kHz in on our test PCs.